tests/integration.rs

```diff
@@ -3883,32 +3883,38 @@ mod tests {
             let mut kernel_path = workload_path.clone();
             kernel_path.push("vmlinux");
 
+            let mut pvh_kernel_path = workload_path.clone();
+            pvh_kernel_path.push("vmlinux.pvh");
+
             let mut initramfs_path = workload_path;
             initramfs_path.push("alpine_initramfs.img");
 
             let test_string = String::from("axz34i9rylotd8n50wbv6kcj7f2qushme1pg");
             let cmdline = format!("console=hvc0 quiet TEST_STRING={}", test_string);
 
-            let mut child = GuestCommand::new(&guest)
-                .args(&["--kernel", kernel_path.to_str().unwrap()])
-                .args(&["--initramfs", initramfs_path.to_str().unwrap()])
-                .args(&["--cmdline", &cmdline])
-                .capture_output()
-                .spawn()
-                .unwrap();
-
-            thread::sleep(std::time::Duration::new(20, 0));
+            vec![kernel_path, pvh_kernel_path]
+                .iter()
+                .for_each(|k_path| {
+                    let mut child = GuestCommand::new(&guest)
+                        .args(&["--kernel", k_path.to_str().unwrap()])
+                        .args(&["--initramfs", initramfs_path.to_str().unwrap()])
+                        .args(&["--cmdline", &cmdline])
+                        .capture_output()
+                        .spawn()
+                        .unwrap();
 
-            let _ = child.kill();
-            match child.wait_with_output() {
-                Ok(out) => {
-                    let s = String::from_utf8_lossy(&out.stdout);
-                    println!("{}", s);
-                    aver_ne!(tb, s.lines().position(|line| line == test_string), None);
-                }
-                Err(_) => aver!(tb, false),
-            }
+                    thread::sleep(std::time::Duration::new(20, 0));
 
+                    let _ = child.kill();
+                    match child.wait_with_output() {
+                        Ok(out) => {
+                            let s = String::from_utf8_lossy(&out.stdout);
+                            println!("{}", s);
```

Comment:
Leftover from debugging?

Reply:
Yes, looks like it... That print was present in the original code, but I didn't catch it when moving the block inside the loop. I'll remove it.

Reply:
Thanks @aljimenezb :)